### PR TITLE
[BREAKING] add Routes schema; remove Transfer.type

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: schemas/Routes.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "description": "a list of routes from a connector",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "source_ledger": {"$ref": "Iri.json"},
+      "destination_ledger": {"$ref": "Iri.json"},
+      "connector": {"$ref": "Iri.json"},
+      "min_message_window": {"type": "number", "minimum": 0, "description": "seconds"},
+      "source_account": {"$ref": "Iri.json"},
```
Comment (emschwartz) [inline schemas/Routes.json]: Might be helpful to add descriptions to these properties so it's clear what they're for

**Comment by sentientwaffle** [comment]
Ok, added some descriptions.

File: schemas/Routes.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        "type": "array",
+        "minItems": 1,
+        "items": {
+          "type": "array",
+          "minItems": 2,
+          "maxItems": 2,
+          "items": {"type": "number"},
+          "description": "[source_amount, destination_amount]"
+        },
+        "description": "a list of points"
```
Comment (clark800) [inline schemas/Routes.json]: Could this be expanded a bit?

**Comment by clark800** [comment]
LGTM

**Pull request merged**